src/emulator/storage/metadata.ts

```diff
@@ -132,10 +132,15 @@ export class StoredFileMetadata {
   }
 
   private setDownloadTokensFromCustomMetadata() {
-    if (!this.customMetadata) return;
+    if (!this.customMetadata) {
+      return;
+    }
 
     if (this.customMetadata.firebaseStorageDownloadTokens) {
-      this.downloadTokens = this.customMetadata.firebaseStorageDownloadTokens.split(",");
+      this.downloadTokens = [
```

Comment:
This is the part I am unsure about ... but it feels right to me.

Reply:
seens very reasonable to me, we must have just missed this when I made the tweaks to download token logic last week